src/backend/aspen/workflows/nextstrain_run/build_config.py

```diff
@@ -144,3 +150,27 @@ def update_subsampling_for_division(subsampling):
     subsampling["group"][
         "query"
     ] = '''--query "(division == '{division}') & (country == '{country}')"'''  # Keep the country filter in case of multiple divisions worldwide
+
+
+def apply_filters(config, subsampling, template_args):
```

Comment:
@danrlu We run this for overview & non contextualized builds, it supports pango lineages and start/end date filters